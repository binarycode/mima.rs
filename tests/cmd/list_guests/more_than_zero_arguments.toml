args = 'list-guests one'
status.code = 2
