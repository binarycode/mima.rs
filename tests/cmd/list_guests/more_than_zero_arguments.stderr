error: unexpected argument 'one' found

Usage: mima list-guests

For more information, try '--help'.
